docs/kyma-environment-broker/02-03-cls-integration.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Cloud Logging Service Integration
+type: Architecture
+---
+
+The Cloud Logging service (CLS) is a managed logging service for shipping logs from various runtimes such as SKR. It uses the Elastic Stack (open source logging platform Elasticsearch, Fluentd, Kibana) to store, parse, and visualize the application log data coming from Kubernetes, Cloud Foundry applications or any other sources.
+
+ Kyma Environment Broker (KEB) provisions a CLS instance for a global account and creates a binding per SKR cluster. It passes the CLS credentials to the provisioner as SKR overrides, so that Fluent Bit in the SKR is properly configured to push the logs to the CLS instance. The following architecture diagram explains the complete process:
```

Comment:
```suggestion
The Kyma Environment Broker (KEB) provisions a CLS instance for a global account and creates a binding per SKR cluster. It passes the CLS credentials to the provisioner as SKR overrides, so that Fluent Bit in the SKR is properly configured to push the logs to the CLS instance. The following architecture diagram explains the complete process:
```